test/src/test/java/org/corfudb/infrastructure/logreplication/MetadataManagerTest.java

```diff
@@ -37,6 +39,10 @@ public class MetadataManagerTest extends AbstractViewTest {
     public void setUp() {
         corfuRuntime = getDefaultRuntime();
         replicationConfig = Mockito.mock(LogReplicationConfig.class);
+        // Mocking steps for initializing LogEntryWriter in following tests.
+        LogReplicationConfigManager mockConfigManager = Mockito.mock(LogReplicationConfigManager.class);
+        when(replicationConfig.getConfigManager()).thenReturn(mockConfigManager);
```

Comment:
consider using Mockito.when()... instead of importing statically, to keep it uniform with other usages in this class